.circleci/config.yml

```diff
@@ -565,6 +565,7 @@ jobs:
       - *upload-coverage
 
   jest:
+    resource_class: medium+
```

Comment:
Do we have budget approval for this?

Reply:
@thomasmaas @gsaslis what do you reckon?

Reply:
Let’s just do it, we can always undo it

Reply:
Apparently a support ticket has to opened, in order for this to work.

Reply:
It LGTM to merge but let's be sure it won't be for nothing.

Reply:
yup just go ahead. we already have the option to use different resource classes enabled. (and we do use it already in porta ci). 

btw, this won't necessarily cost _that much more_ because if jest takes advantage of the extra cores and the test finish "faster" (and billing is per minute), it won't be "double" the cost. 

tl;dr go go go go go go go go go